evaluation/utils/plot_data.R

```diff
@@ -0,0 +1,18 @@
+# Packages ----------------------------------------------------------------
+library(ggplot2)
+library(cowplot)
+library(dplyr)
+
+plot_data <- function(weekly, daily){
+plot <- weekly %>%
+    mutate(state_location = paste0(state, " (", location, ")")) %>%
```

Comment:
Because the forecasts use only location codes, but the plots only use state names. so sometimes I find myself going back and forth between the forecasts and the plots & I find it annoying to have to go via a state-to-location-code lookup table. adding in the code to the plots to skip that - just a personal preference though!

Reply:
ah right got you. Well it doesn't really harm anything so might be a good idea.